<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetAssist</class>
 <widget class="QWidget" name="NetAssist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="send">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>510</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="sendEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>621</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="receiveEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>690</width>
     <height>311</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>远程地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>协议类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>本地端口</string>
   </property>
  </widget>
  <widget class="QComboBox" name="protocol">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>TCP Client</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TCP Server</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UDP</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>远程端口</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="remoteaddr">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>输入域名或IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="colon">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>：</string>
   </property>
  </widget>
  <widget class="QLabel" name="colon_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>：</string>
   </property>
  </widget>
  <widget class="QLabel" name="colon_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>：</string>
   </property>
  </widget>
  <widget class="QLabel" name="colon_4">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="receivegroup">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>120</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>接收设置</string>
   </property>
   <widget class="QCheckBox" name="autonewline">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>自动换行</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="receiveHex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>显示hex</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="islogmode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>日志模式</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="receivenoshow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>接收不显示</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="clearsend">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清除发送区</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="sendgroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>120</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>发送设置</string>
   </property>
   <widget class="QCheckBox" name="atnewline">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>AT指令自动
加回车换行</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sendHex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送hex</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="clearreceive">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清除接收区</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startclose">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>客户端</string>
   </property>
  </widget>
  <widget class="QLabel" name="colon_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="clientlist">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="remoteport">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>8080</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="localport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>8443</number>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnclient">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>断开连接</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
